src/CoreBundle/Controller/SecurityController.php

```diff
@@ -52,4 +68,25 @@ public function loginJson(AuthenticationUtils $authenticationUtils): Response
 
         return new JsonResponse($data, Response::HTTP_OK, [], true);
     }
+
+    /**
+     * @Route("/login_failed", name="login_failed")
+     */
+    public function loginFailed(Request $request): JsonResponse
```

Comment:
El trackELoginRecord con success = false se puede registrar en el mismo loginJson dentro de la primera condición (`if (!$this->isGranted('IS_AUTHENTICATED_FULLY'))`), lo que ahorraría una request más cuando el login falla